src/components/ViewList.js

```diff
@@ -123,7 +138,11 @@ let mapDispatchToProps = dispatch => {
     },
     resetFilter: () => {
       dispatch(updateFilter("All"));
-    }
+    },
+    targetEdit: (target) => {
+      dispatch(updateInput("companyEditSelect", target));
+
+    },
```

Comment:
Set companyEditSelect to clicked target -- sets Redux to load its data into edit form.